runtime/src/bank.rs

```diff
@@ -3704,18 +3702,17 @@ impl Bank {
         }
     }
 
-    /// Returns the accounts, sorted by pubkey, that were part of accounts delta hash calculation
+    /// Returns the accounts, sorted by pubkey, that were part of accounts lt hash calculation
```

Comment:
nit: Would more general phrasing be appropriate given that rent is gone now too ?
> Returns the accounts, sorted by pubkey, that were part of this bank